### DISCUSSION TITLE
Enabling debug logging?

### DISCUSSION THREAD

**Discussion by disser** [scambier/obsidian-omnisearch#183] [General]
Is there any option to enable logging?  One of my vaults crashes on indexing and I'd like to submit a bug report but want to figure out more detail first.  The error I'm getting is `Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'path')` but I'm trying to figure out if it's a particular file causing the error.  I'm using 1.10.0 on the latest insider build.

**Comment by scambier** [comment]
There isn't an option but I definitely should add one. Your bug is also maybe fixed with https://github.com/scambier/obsidian-omnisearch/issues/181#issuecomment-1385886777

**Reply by disser** [reply]
Thank you! That fix does resolve the issue.
